lib/nexmo_jwt.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
Please revert the name change of all the files bac to using a `-` and not a `_`. Would be happy to consider this change in a separate PR.